sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -556,3 +557,65 @@ def test_no_feature_selected():
         X_selected = assert_warns_message(
             UserWarning, 'No features were selected', selector.transform, X)
         assert_equal(X_selected.shape, (40, 0))
+
+
+def test_mutual_info_classification():
```

Comment:
can you speed up the test?

this single test takes about 50% of the test fime of the whole file